google_searchconsole/gsc_bq_dta_dga.py

```diff
@@ -0,0 +1,90 @@
+import datetime
+import re
+
+import pandas as pd
+
+from google.cloud import bigquery
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+from oauth2client.service_account import ServiceAccountCredentials
+
+import parameters as pe
```

Comment:
As above, provide a template for parameters. 

SERVICE_ACCOUNT_FILE might be better entered as a command line parameter or environment variable with a default value of "credentials.json" or something like that if it is not provided.

Reply:
parameter template file created for upload